### PR TITLE
Add reference page for counting (DNF Count Me)

### PR DESCRIPTION
**Pull request by travier**
Includes instructions for disabling counting.

### CODE REVIEW

File: modules/ROOT/pages/counting.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+= Node counting
+
+Starting with the first releases in August, Fedora CoreOS nodes will be counted by the Fedora infrastructure via the Count Me feature. This system is explicitly designed to make sure that no personally identifiable information is sent from counted systems. It also insures that the Fedora infrastructure does not collect any personal data. The nickname for this counting mechanism is "Count Me", from the option name. Implementation details of this feature are available in https://fedoraproject.org/wiki/Changes/DNF_Better_Counting[DNF Better Counting change request for Fedora 32]. In short, the Count Me mechanism works by telling Fedora servers how old your system is (with a very large approximation).
```
Comment (dustymabe) [inline modules/ROOT/pages/counting.adoc:3]:
```suggestion
Starting with the first releases in August, Fedora CoreOS nodes will be counted by the Fedora infrastructure via the Count Me feature. This system is explicitly designed to make sure that no personally identifiable information is sent from counted systems. It also ensures that the Fedora infrastructure does not collect any personal data. The nickname for this counting mechanism is "Count Me", from the option name. Implementation details of this feature are available in https://fedoraproject.org/wiki/Changes/DNF_Better_Counting[DNF Better Counting change request for Fedora 32]. In short, the Count Me mechanism works by telling Fedora servers how old your system is (with a very large approximation).
```

I'm bad with ensure vs insure, do we need to change it?

File: modules/ROOT/pages/counting.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+On Fedora CoreOS nodes, this functionnality is implemented in https://coreos.github.io/rpm-ostree/countme/[rpm-ostree as a stand-alone method]. The new implementation has the same privacy preserving properties as the original DNF implementation.
```
Comment (dustymabe) [inline modules/ROOT/pages/counting.adoc:5]:
```suggestion
On Fedora CoreOS nodes, this functionality is implemented in https://coreos.github.io/rpm-ostree/countme/[rpm-ostree as a stand-alone method]. The new implementation has the same privacy preserving properties as the original DNF implementation.
```

**Review by dustymabe — APPROVED** [review]
LGTM

**Review by jlebon — APPROVED** [review]
Nice!

**Pull request merged**